cocos/renderer/ccShader_3D_Skybox.vert

```diff
@@ -1,12 +1,13 @@
 const char* cc3D_Skybox_vert = STRINGIFY(
 
-uniform float u_scalef;
+uniform mat4  u_cameraRot;
 attribute vec3 a_position;
 varying vec3 v_reflect;
 
```

Comment:
It seems that Model,View,Project matrix are not considered in this shader. How to specify vertex data for Sky box?
Besides of this, it seems the uniform `u_cameraRot` will take scale into consideration if there is scale in `Camera's transform`.